### PR TITLE
add animations folder with first animation mixins

### PR DESCRIPTION
**Pull request by hippogriffic**
general transition mixin to add simple transitions. Expand and shrink mixins for expand and hiding content

these have been tested locally using angular `ng-hide` classes

@incuna/frontend please review

### CODE REVIEW

**Comment by pandalion** [comment]
@hippogriffic do we get a gif demo? :D

File: bower.json [hunk 1]
```diff
@@ -7,6 +7,9 @@
   ],
   "description": "Incuna's Transitions Library",
   "license": "MIT",
+  "dependencies": {
+    "bourbon": "~4.0.0"
```
Comment (pandalion) [inline bower.json]: should we fix the version?

**Comment by hippogriffic** [comment]
oh, yeah I can do that... after lunch though :D

**Comment by hippogriffic** [comment]
blah

blah

blah

@pandalion gifs for your :panda_face:

**Comment by Minglee01** [comment]
Looks cool.

**Pull request merged**